plugins/modules/zypper_repository_info.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2024, Tobias Zeuch <tobias.zeuch@sap.com>
+#
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: zypper_repository_info
+author: "Tobias Zeuch (@TobiasZeuch181)"
+version_added: 9.5.0
```

Comment:
Hi @TobiasZeuch181 , version 9.5.0 has been released, we must update this:
```suggestion
version_added: 10.0.0
```